### DISCUSSION TITLE
Detect new fans installed?

### DISCUSSION THREAD

**Discussion by Suhycide** [Rem0o/FanControl.Releases#1025] [Q&A]
I have an issue, I just put 3 brand new Corsair SP120 RGB Elite fans in my system and the software wont detect them, it just runs the sensor thing and reverts to how it was with my old 2 intake fans instead of my new 3. Do i need to uninstall and reinstall, and make new curves? If so, how do i properly uninstall this application and reinstall it? I cant seem to find it in my file explorer, or im just stupid lol

**Comment by Rem0o** [comment]
FanControl doesn't detect "fans", it shows "fan headers" from your motherboard. You need to plug your fans to motherboard headers, and use those headers from fancontrol to control these fans, if that make sense.

**Reply by Suhycide** [reply]
i have all 6 of my fans hooked to the Commander XT, which is detected by the software, im just not sure how to tell if the new fans are being controlled by it

**Reply by Suhycide** [reply]
the only fan on a header is the Wraith Prism cooler for my R9 3900X and is set to my cpu curve

**Reply by Rem0o** [reply]
Wait, how is the Commander XT detected? It's not even supported.
